sql/wsrep_var.cc

```diff
@@ -281,7 +283,9 @@ void wsrep_start_position_init(const char *val) {
     return;
   }
 
-  wsrep_set_local_position(NULL, val, strlen(val), false);
+  if (wsrep_set_local_position(NULL, val, strlen(val), false)) {
+    WSREP_ERROR("wsrep_set_local_position() failed");
```

Comment:
I am not sure if it is a good idea to put a function name in the error message. 

Can we use something like `Server failed to set local position.` ? 
If we want to write the function name here, then we could consider `WSREP_DEBUG()` here.